docs/moduleAna5Utils.rst

```diff
@@ -0,0 +1,105 @@
+
+#####################################################################
+ana5Utils
+#####################################################################
+
+
+distanceTimeAnalysis: Visualizing the temporal evolution of flow variables
+------------------------------------------------------------------------------
+
+With the functions gathered in this module, flow variables of avalanche simulation results can be
+visualized in a distance versus time diagram, the so called **thalweg-time diagram**.
+The **tt-diagram** provides a way to identify main features of the temporal evolution of
+flow variables along the :term:`avalanche path`.
+This is based on the ideas presented in :cite:`FiFrGaSo2013` and :cite:`RaKo2020`, where
+avalanche simulation results have been transformed into the radar coordinate system to facilitate
+direct comparison, combined with the attempt to analyze simulation results in an avalanche path
+dependent coordinate system (:cite:`Fi2013`).
+In addition to the **tt-diagram**, :py:mod:`ana5RangeTime` also offers the possibility to
+produce simulated **range-time diagrams** of the flow variables with respect to a radar field
+of view. With this, simulation results can be directly compared to radar measurements in terms
```

Comment:
after radar measurements add (for example MTI images from [ref])

ref = https://doi.org/10.1002/2017JF004375

Reply:
added